operations/iscsi_sbps/Node_Personalization_Verification.md

```diff
@@ -0,0 +1,256 @@
+  |     | o- lun6 .. [fileio/602f57d78f7162d (/var/lib/cps-local/boot-images/PE/CPE-intel.x86_64-25.03.squashfs) (default_tg_pt_gp)]
+  |     | o- lun7  [fileio/31fcfe145252d40 (/var/lib/cps-local/boot-images/PE/CPE-intel.x86_64-25.09-20250701.squashfs) (default_tg_pt_gp)]
+  |     | o- lun8 . [fileio/2b9d303c515911a (/var/lib/cps-local/boot-images/PE/CPE-nvidia.x86_64-25.03.squashfs) (default_tg_pt_gp)]
+  |     | o- lun9  [fileio/162eec8935d9251 (/var/lib/cps-local/boot-images/PE/CPE-nvidia.x86_64-25.09-20250701.squashfs) (default_tg_pt_gp)]
+  |     | o- lun10  [fileio/7df1b36a366a1a3 (/var/lib/cps-local/boot-images/83f3f34c-b09c-445b-8253-f0708f588328/rootfs) (default_tg_pt_gp)]
+  |     | o- lun11  [fileio/8767f2cfdb49d5f (/var/lib/cps-local/boot-images/efd2f545-1a08-4f63-bd18-45c2bfcf7efc/rootfs) (default_tg_pt_gp)]
+  |     | o- lun12  [fileio/d4b0cd14d4625db (/var/lib/cps-local/boot-images/50256d2c-60a8-41f8-ab37-901515d607e8/rootfs) (default_tg_pt_gp)]
+  |     | o- lun13  [fileio/a78fe7060cf474d (/var/lib/cps-local/boot-images/5095f83f-b500-4f7c-81be-76b185ad7108/rootfs) (default_tg_pt_gp)]
+  |     o- portals .................................................................................................... [Portals: 2]
+  |       o- 10.252.1.8:3260 .................................................................................................. [OK]
+  |       o- 10.253.0.2:3260 .................................................................................................. [OK]
+  o- loopback ......................................................................................................... [Targets: 0]
+  o- srpt ............................................................................................................. [Targets: 0]
+  o- vhost ............................................................................................................ [Targets: 0]
+  o- xen-pvscsi ....................................................................................................... [Targets: 0]
+    ```
+
+   From the example `targetcli ls` output, we can see that the `fileio` backing store for the PE image `CPE-intel.x86_64-25.03.squashfs`
+   is created i.e `/var/lib/cps-local/boot-images/PE/CPE-intel.x86_64-25.03.squashfs` and corresponding iSCSI `LUN` i.e `lun6` is created.
+
+   Similarly, for the `rootfs` image with ID `50256d2c-60a8-41f8-ab37-901515d607e8`, `fileio` backing store `/var/lib/cps-local/boot-images/50256d2c-60a8-41f8-ab37-901515d607e8/rootfs` is created and corresponding iSCSI `LUN` i.e `lun12` is created with the matching `rootfs` ID.  
+   
+    The portals in the above example `targetcli ls` output are `10.252.1.8:3260` and `10.253.0.2:3260`. These portals are combination 
+    HSN/NMN IP and the protocol number of iSCSI i.e `3260`. We can check each of these portals correspond to HSN/NMN interface as below: 
+
+   ```text
+    ncn-w002:~ # ip addr | grep hsn
```

Comment:
Commands should not be embedded inside the text block. Commands go in their own blocks, and the output goes in a separate block.